server.go

```diff
@@ -269,6 +271,12 @@ func (s *Server) RunUntilShutdown(mainErrChan <-chan error) error {
 	serverOpts = append(serverOpts, rpcServerOpts...)
 	serverOpts = append(serverOpts, ServerMaxMsgReceiveSize)
 
+	keepAliveParams := keepalive.ServerParameters{
+		MaxConnectionIdle: time.Minute * 2,
```

Comment:
I expected to see smth like 15 - 30 sec here. Is this like a widely accepted default-ish value?

Reply:
Just kinda chose an initial value tbh...